code/Proxy/ConfigManifestProxy.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace LeKoala\DebugBar\Proxy;
+
+use SilverStripe\Config\Collections\CachedConfigCollection;
+
+class ConfigManifestProxy extends CachedConfigCollection
+{
+    /**
+     * @var CachedConfigCollection
+     */
+    protected $parent;
+
+    /**
+     * @var array
+     */
+    protected static $configCalls = [];
+
+    /**
+     * @param ConfigCollectionInterface $parent
+     */
+    public function __construct(CachedConfigCollection $parent)
```

Comment:
I see what you're saying. Should we then update the PHPDoc parameter to reflect CachedConficCollection and not the abstraction instead?